package.json

```diff
@@ -21,7 +21,8 @@
     "generate": "./generate-client.sh",
     "watch": "tsc --watch",
     "test": "nyc mocha",
-    "prepare": "npm run build"
+    "prepare": "npm run build",
+    "docs": "typedoc src/gen/api/coreV1Api.ts"
```

Comment:
Can you extend this to add docs for other/all parts of the library? Does typedoc support a wild-card?

Reply:
I've swapped it to `src/gen/api`. Are there any other directories that would be useful to have typedocs generated?